recipe/build.sh

```diff
@@ -4,6 +4,10 @@ echo "=== Building ${PKG_NAME} (py: ${PY_VER}) ==="
 
 set -ex
 
+echo "####################################################################"
+echo "Building PyTorch using BLAS implementation: $blas_impl              "
+echo "####################################################################"
```

Comment:
Nit: could we combine this with the message above somehow, so we get a single big message with all important details?

Reply:
done